src/StatusQ/Controls/StatusChatInfoButton.qml

```diff
@@ -46,191 +40,158 @@ Rectangle {
-                    y: parent.height + 12
-                }
-            }
-        }
+                StatusIcon {
+                    objectName: "mutedIcon"
+                    Layout.preferredWidth: visible ? 13 : 0
+                    Layout.preferredHeight: visible ? 13 : 0
+                    icon: "tiny/muted"
+                    color: mutedIconSensor.containsMouse ? Theme.palette.directColor1 : Theme.palette.baseColor1
+                    visible: root.muted
+
+                    MouseArea {
+                        id: mutedIconSensor
+                        hoverEnabled: true
+                        cursorShape: Qt.PointingHandCursor
+                        anchors.fill: parent
+                        onClicked: root.unmute()
+                    }
 
-        Item {
-            id: statusChatInfoButtonSubTitle
-            anchors.left: statusChatInfoButtonTitle.left
-            anchors.top: statusChatInfoButtonTitle.bottom
-            visible: !!statusChatInfoButton.subTitle || statusChatInfoButton.pinnedMessagesCount > 0
-            height: visible ? chatType.height : 0
-            width: Math.min(parent.width - statusChatInfoButtonTitle.anchors.leftMargin
-                            - identicon.width - identicon.anchors.leftMargin - 8,
-                            implicitWidth)
-
-            implicitWidth: chatType.implicitWidth + pinIconDelta + 8
-
-
-            property real pinIconDelta: pinIcon.visible ? pinIcon.width + pinIcon.anchors.leftMargin
-                                                          + divider.width + divider.anchors.leftMargin
-                                                        : 0
-
-            StatusSelectableText {
-                id: chatType
-                text: statusChatInfoButton.subTitle
-                color: Theme.palette.baseColor1
-                font.pixelSize: 12
-                width: Math.min(parent.width - (pinIcon.visible ? divider.width + divider.anchors.leftMargin + pinIcon.width + pinIcon.anchors.leftMargin : 0),
-                                implicitWidth)
-                onLinkActivated: statusChatInfoButton.linkActivated(link)
+                    StatusToolTip {
+                        id: statusToolTip
+                        text: qsTr("Unmute")
+                        visible: mutedIconSensor.containsMouse
+                        orientation: StatusToolTip.Orientation.Bottom
+                        y: parent.height + 12
+                    }
+                }
             }
 
-            Rectangle {
-                id: divider
-                height: 12
-                width: 1
-                color: Theme.palette.directColor7
-                anchors.left: chatType.right
-                anchors.leftMargin: 4
-                anchors.verticalCenter: chatType.verticalCenter
-                visible: !!chatType.text && pinIcon.visible
-            }
+            // subtitle
```

Comment:
Since comments can get outdated very soon I would prefer to have `id: subtitle` even if it is not used.